nyaa/templates/help.html

```diff
@@ -11,8 +11,11 @@ <h3 id="{{ label }}">{{ heading }}<a class="header-anchor" href="#{{ label }}"><
 {{ linkable_header("How to Download", "how-to-dl") }}
 <div>
 	This site only offers torrent files and magnet links. To download the content of
-	the torrents, you will have to use a BitTorrent client such as qBitTorrent,
-	µTorrent, Transmission or Deluge.
+	the torrents, you will have to use a BitTorrent client such as 
+	<a href="https://www.qbittorrent.org/download.php">qBitTorrent</a>, 
+	<a href="https://transmissionbt.com/download/">Transmission</a>, 
+	<a href="http://dev.deluge-torrent.org/wiki/Download">Deluge</a> or 
+	<a href="http://www.utorrent.com/downloads">µTorrent</a>.
```

Comment:
FYI: µTorrent is [cancer](https://www.reddit.com/r/torrents/comments/6ao652/warning_a_utorrent_ad_is_using_a_flash_exploit_to/). Full of adware. I wouldn't add it to the list. If that's not possible, tell them to find a µTorrent v2.X installer, which is from 2011.